### PR TITLE
Added SSL Troubleshooting & Fixed Spelling Errors

### PR DESCRIPTION
**Pull request by DiscoverSquishy**
Added a troubleshooting section for installing/renewing SSL's.

Fixed spelling in a couple files.

### CODE REVIEW

File: tutorials/creating_ssl_certificates.md [hunk 1]
```diff
@@ -34,3 +34,25 @@ which provides more options, and is much more powerful than certbot.
...
+If you get a `Insecure Connection` or related when trying to access your panel, it is likely that the SSL has expired.
+This can be easily fixed by renewing the SSL certificate, although using the command
+``` text
+certbot renew
+```
+Wont do the job. As it'll give a error such as `Error: Attempting to renew cert (domain) from /etc/letsencrypt/renew/domain.conf produced an unexpected error`.
+This will happen especially if your running NGINX instead of Apache, the solution for this is to stop NGINX, renew the certificate then start NGINX once again.
+You can do this by running,
+```bash
+systemctl stop NGINX
```
Comment (parkervcp) [inline tutorials/creating_ssl_certificates.md]: nginx is all lowercase in context.

File: tutorials/creating_ssl_certificates.md [hunk 2]
```diff
@@ -34,3 +34,25 @@ which provides more options, and is much more powerful than certbot.
...
+```
+To stop the NGINX service. Now you need to renew the certificate, you can do this by running,
+```bash
+certbot renew
+```
+Wait for the process to complete. 
+Once it has, you may now re-start the NGINX service using,
+```bash
+systemctl start NGINX
```
Comment (parkervcp) [inline tutorials/creating_ssl_certificates.md]: nginx is all lowercase in context.

File: tutorials/creating_ssl_certificates.md (hunk 1 above)
Comment (DiscoverSquishy) [reply tutorials/creating_ssl_certificates.md]: I have lowercased it.

File: tutorials/creating_ssl_certificates.md (hunk 2 above)
Comment (DiscoverSquishy) [reply tutorials/creating_ssl_certificates.md]: I have lowercased it.

**Comment by DiscoverSquishy** [comment]
@parkervcp It's done. Is there anything else?

**Pull request merged**